### ISSUE TITLE
Is there any way to change the sprite of the declared player?

### ISSUE THREAD

**Issue by Fixid-Fuzz** [RafaelBarbosatec/bonfire#265]
Dear developer.
I am making RPG games with the precious engine you made.

I want to make an RPG that equips items.
But I don't know how to modify the sprite of player declared in BonfireTiledWidget.

Is there any way to change the sprite already declared in BonfireTiledWidget?

If it already exists, can you tell me how to do it?

thanks for reading my question.

**Comment by RafaelBarbosatec** [comment]
Hi @Fixid-Fuzz !
Thanks! 😊
You can use the method `replaceAnimation`. With this, you can change the SimpleDirectionAnimation.
Bonfire also have methods to merge images, this will can help you to build dynamics SpriteSheets with differents clothes, equipments, etc. I use this in SunnyPlace project.

**Comment by RafaelBarbosatec** [comment]
I'll publish the example about this in this repository

**Comment by Fixid-Fuzz** [comment]
Thank you so much!!!

**Comment by Fixid-Fuzz** [comment]
> 안녕@Fixid-Fuzz! 감사!😊 방법을 사용할 수 있습니다 `replaceAnimation`. 이를 통해 SimpleDirectionAnimation을 변경할 수 있습니다. Bonfire에는 이미지를 병합하는 방법도 있습니다. 이것은 다른 옷, 장비 등으로 역학 SpriteSheets를 구축하는 데 도움이 될 것입니다. 저는 이것을 SunnyPlace 프로젝트에서 사용합니다.
> 
> https://bonfire-engine.github.io/#/loading_assets?id=mergin-images

https://user-images.githubusercontent.com/99014005/175860275-2e1a2712-8739-4354-94b7-de719895931e.mov

Dear Developer
I wrote the code the way you suggested.
The character changes but is not visible until it moves.

When constructionMode is set to true, there is no sprite, but the area is held.

This is the code I wrote. Is there anything else I need to write?

------------------------------------------------------------------------
final localPlayer = BonfireInjector().get<LocalPlayerController>();
await localPlayer.component!.replaceAnimation(TestSpriteSheet.impAnimations);
------------------------------------------------------------------------
